### PR TITLE
Asciiquarium

### PR DESCRIPTION
**Pull request by the-maldridge**
One of my favorite useless packages, this draws an aquarium drawn entirely from ascii characters on your terminal.

### CODE REVIEW

File: srcpkgs/asciiquarium/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'asciiquarium'
+pkgname=asciiquarium
+version=1.1
+revision=1
+wrksrc=${pkgname}_${version}
+depends="perl perl-Term-Animation"
+short_desc=" Enjoy the mysteries of the sea from the safety of your own terminal"
```
Comment (Vaelatern) [inline srcpkgs/asciiquarium/template]: Please remove space at beginning of short_desc.

File: srcpkgs/asciiquarium/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+maintainer="Michael Aldridge <aldridge.mac@gmail.com>"
+license="GPL-2"
+homepage="http://www.robobunny.com/projects/asciiquarium/html/"
+distfiles="http://www.robobunny.com/projects/asciiquarium/asciiquarium.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/asciiquarium/template]: Please use the versioned tarball, `http://www.robobunny.com/projects/asciiquarium/asciiquarium_1.1.tar.gz`

File: srcpkgs/perl-Term-Animation/template [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+pkgname=perl-Term-Animation
+version=2.6
+revision=1
+wrksrc=${pkgname//perl-//}-${version}
+build_style=perl-module
+hostmakedepends="perl"
+short_desc="Terminal Animation module for perl"
+maintainer="Michael Aldridge <aldridge.mac@gmail.com>"
+license="Perl"
+homepage="http://robobunny.com/projects/animation/haml/"
```
Comment (Vaelatern) [inline srcpkgs/perl-Term-Animation/template]: Recommend the homepage is not 404: `s/haml/html/` perhaps

**Comment by Vaelatern** [comment]
Please make the perl module depend on perl, and thus remove the `perl` dependency from `asciiquarium`.

**Comment by the-maldridge** [comment]
I agree that the library should depend on perl, however I do not agree that asciiquarium itself shouldn't depend on perl.  From the manual the depends field is specifically for runtime dependencies that are otherwise not picked up by xbps.  Since asciiquarium has a direct dependancy on perl, it should therefore have that in that field.

**Comment by Vaelatern** [comment]
And, finally, both need `noarch=yes`

**Pull request merged**